core/network/fabric-interop-cc/makefile

```diff
@@ -1,4 +1,7 @@
-include makefile.variables.docker
+DOCKER_IMAGE_NAME ?= weaver-fabric-cc
+DOCKER_TAG ?= `cat contracts/interop/VERSION`
```

Comment:
Should this be the version for `contracts/interop` or a separate one specified in this folder?

Reply:
There are multiple modules now in `fabric-interop-cc` folder, so `contracts/interop` has a separate file `VERSION` to specify its version, we just use that file here for Docker Tag, because this builds image just for `contracts/interop`